### PR TITLE
faq: readme: Add entry for calling op-tee from android .apk apps

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

File: faq/README.md [hunk 1]
```diff
@@ -361,6 +361,24 @@ Those can only be shared are under NDA with Google and Microsoft. Linaro can
 help members of Linaro to get access to those. As of now, we cannot share it
 with non-members.
 
+### How can I use or call OP-TEE (GlobalPlatform) APIs from an Android (.apk)
+app?
```
Comment (jbech-linaro) [inline faq/README.md]: Should be on previous row/line?

File: faq/README.md [hunk 2]
```diff
@@ -361,6 +361,24 @@ Those can only be shared are under NDA with Google and Microsoft. Linaro can
...
+Use JNI. First get familiar with
+https://developer.android.com/ndk/samples/sample_hellojni.html and make sure
```
Comment (jbech-linaro) [inline faq/README.md]: Make link instead, i.e., `[sample_hellojni.html](https://developer.android.com/ndk/samples/sample_hellojni.html)`

File: faq/README.md [hunk 3]
```diff
@@ -361,6 +361,24 @@ Those can only be shared are under NDA with Google and Microsoft. Linaro can
...
+you can run the example. After that, replace the C-side Implementation with say
+https://github.com/linaro-swg/hello_world or one of the examples in
```
Comment (jbech-linaro) [inline faq/README.md]: Same here, I prefer (short) links for both hello_world and optee_example.

**Comment by jbech-linaro** [comment]
Thanks for adding this Victor, two things here ... I believe @shovanuk are pulling this file from our https://github.com/OP-TEE/build/blob/master/faq.md (which is the true source)? I would like to avoid (manually) maintain two FAQ's. We must decide how we want to deal with this.

**Comment by shovanuk** [comment]
@ jbech-linaro yes changes in this repo directly reflect the content on the website. If it's not a problem, I would suggest keeping all the website content in this repo.

**Comment by jbech-linaro** [comment]
> @ jbech-linaro yes changes in this repo directly reflect the content on the website. If it's not a problem, I would suggest keeping all the website content in this repo.

That is OK for me, but in that case we shall remove the FAQ.md from out build.git (and all reference to it). Actually it is probably better to keep it there in that case, but wipe the content and just tell people to go to the web-page instead. I think we have linked to the FAQ on build.git from several places and it getting `404's` isn't very fun.

**Comment by vchong** [comment]
You're welcome!
All comments addressed and forced pushed. Hope it's ok.

**Comment by shovanuk** [comment]
@jbech-linaro Agree, have the FAQ file in build.git and just redirect to website repo. Let me know if you are happy for me to merge this request.

**Comment by jbech-linaro** [comment]
Sounds good to me, let's change the FAQ in build.git in the near future to just becomes a redirect (having that said, your PR in build.git @vchong is good for now).

**Comment by vchong** [comment]
@jbech-linaro For https://github.com/OP-TEE/build/blob/master/faq.md, I've not found a way to automatically redirect it to https://github.com/OP-TEE/optee_website/tree/master/faq. Can only zero out the page and add a note that says something like 'This page has been moved [here](https://github.com/OP-TEE/optee_website/tree/master/faq)'.

**Comment by jbech-linaro** [comment]
> Can only zero out the page and add a note that says something like 'This page has been moved here'.

That is exactly what I had in mind :)

**Comment by vchong** [comment]
Done. https://github.com/OP-TEE/build/pull/195
Thanks! :)

**Pull request merged**